spec/unit/messages/users_list_message_spec.rb

```diff
@@ -35,6 +35,21 @@ module VCAP::CloudController
         expect(message.requested?(:origins)).to be_truthy
       end
 
+      context 'partial_usernames is used in .from_params' do
+        let(:params) do
+          {
+            partial_usernames: ['bob'],
+            origins: ['uaa']
+          }
+        end
+
+        it 'is valid' do
+          message = UsersListMessage.from_params(params)
+          expect(message).to be_valid
+          expect(message.partial_usernames).to eq(%w[bob])
+        end
+      end
+
       context 'guids, usernames, origins are nil' do
         let(:params) do
           {
```

Comment:
The `context 'guids, usernames, origins must be arrays'` should not be underneath `context 'guids, usernames, origins are nil'`.

Reply:
We can move the test